src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -121,7 +121,7 @@ public function text($message)
 
         $messages = is_array($message) ? array_values($message) : array($message);
         foreach ($messages as $message) {
-            $this->writeln(sprintf(' %s', $message));
+            $this->writeln(sprintf('%s', $message));
         }
```

Comment:
I suspect this is done on purpose. What does this actually fix? If this is indeed a legit fix, you can remove the sprintf.

Reply:
I also guess this was done on purpose for indentation probably?